main.go

```diff
@@ -37,35 +36,31 @@ func main() {
 
 // Usage returns, you guessed it, the usage information
 func Usage() string {
-	return "An opinionated CLI for the Rackspace cloud"
+	return "Command-line interface to manage Rackspace Cloud resources"
 }
 
 // Desc returns, you guessed it, the description
 func Desc() string {
-	return `Rack is an opinionated command-line tool that allows Rackspace users
-to accomplish tasks in a simple, idiomatic way. It seeks to provide
-flexibility through common Unix practices like piping and composability. All
-commands have been tested against Rackspace's live API.`
+	return `The rack CLI manages authentication, configures a local setup, and provides workflows for operations on Rackspace cloud resources.`
 }
 
 // Cmds returns a list of commands supported by the tool
 func Cmds() []cli.Command {
 	return []cli.Command{
 		{
-			Name: "init",
-			Usage: strings.Join([]string{"For Linux and OS X, creates the `rack` man page and sets up command completion for the Bash shell.",
-				"\tFor Windows, creates a `posh_autocomplete.ps1` file in the `$HOME/.rack` directory. You must run the file to set up command completion."}, "\n"),
+			Name:   "configure",
+			Usage:  "Interactively create a config file for Rackspace authentication.",
+			Action: configure,
+		},
+		{
+			Name:  "init",
+			Usage: "[Linux/OS X only] Creates the rack man page and sets up command completion for the Bash shell.",
```

Comment:
I takes a slice of strings and joins them with a specified string. It is used here to break the string up into individual lines and then join them with a new-line character ("\n").